broker/mosquitto/config/passwd_file

```diff
@@ -0,0 +1 @@
+mosquitto:$6$aB/lZEmj399gcmFN$5F2lAaVaGZ6dDYpvwlv4zONbn2CqwsypZuL0Gy3O5u0SgmzSVUurI7oSO/tTD1vuFaW9NaM6nb6XIJnd65vUEw==
```

Comment:
Is this the password?

Reply:
This is the hash of the default passoword which has been set to `default`. This way the default application has a password, although a deployed solution would have to change this locally of course.